mautic/mautic
# Fix unescaped builder i18n strings

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | ✔️ 
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #3708
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

If the i18n messages used by the `getJsLang` method contains single quotes (which they do in the french locale for instance), the builder cannot be used as the builder's JS crashes upon opening the builder.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Switch to the french locale or add a single quote in one of the strings used by `getJsLang`
2. Open the builder
3. Try to modify a slot
4. Note you cannot select a slot

#### Steps to test this PR:
1. Apply this PR
1. Switch to the french locale or add a single quote in one of the strings used by `getJsLang`
2. Open the builder
3. Try to modify a slot
4. The builder should function properly
